### ISSUE TITLE
Common config file not getting generated.

### ISSUE THREAD

**Issue by muqsit-0010** [gigaherz/ToolBelt#93]
ToolBelt-1.16.3-1.15.2
MC 1.16.5
Forge 36.1.2

I need to disable toolbelt's anvil upgrade recipes for my mopdpack.
Looking at the code I see that there is an anvilUpgrading property in common config.
But, the problem is that the common config never gets generated.

Also, looking at the decompiled code of the 1.15.2 version I can see that registerConfig is never called for common config.

**Comment by gigaherz** [comment]
The 1.15 version never had a common config. Those values are in the server config.

If you have somehow managed to never once deal with serverconfigs in 1.13`, you can find those files in the serverconfig folder in the save directory. Eg:

```
<instance>/saves/<save folder>/serverconfig/toolbelt-server.toml
```

If you are building a modpack, you can copy the serverconfigs folder into defaultconfigs in the root. Those files will be applied to new saves, or existing saves when updating the pack. Eg:

```
<instance>/defaultconfigs/toolbelt-server.toml
```

Just remember to update the defaultconfigs folder after tweaking, before releasing your pack!

**Comment by AbdulMuqsit** [comment]
Wait, you said 1.13?

 I am working with minecraft 1.16.5.
 ToolBelt version 1.16.3-1.15.2

I had already checked the server config before creating the issue. It does not have an anvilUpgrading property.

This is the server config file that gets generated.

[toolbelt-server.zip](https://github.com/gigaherz/ToolBelt/files/6216051/toolbelt-server.zip)

**Comment by gigaherz** [comment]
Oh my bad, I saw you mention 1.15.2 and assumed that was the mc version you used. 🤦

**Comment by gigaherz** [comment]
So after looking at the 1.16 version, I do have this fixed, but it may be I never released an update with that fix... I'll look into it.

**Comment by AbdulMuqsit** [comment]
That will be great. Thank you

**Comment by gigaherz** [comment]
Should be fixed in v1.16.0
